src/shared/Services/CacheService.cs

```diff
@@ -0,0 +1,38 @@
+using Microsoft.Extensions.Caching.Memory;
+
+namespace dig;
+
+public class CacheService(IMemoryCache memoryCache,
+                            FileCacheService fileCache,
+                            ILogger<CacheService> logger)
+{
+    private readonly IMemoryCache _memoryCache = memoryCache;
+    private readonly FileCacheService _fileCache = fileCache;
```

Comment:
Instead of mounting FileCacheService should we mount IObjectCacheService that way we can swap it for redis easily just by adding a config? Should we wait for another PR to do that?

Reply:
yeah - i want to get this done, and then we can make it so we can swap in other cache providers